content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -43,38 +33,56 @@ located above the project table.
 The table contains buttons to specify filters and other preferences regarding
 which commits and columns to display.
 
-![](https://static.iterative.ai/img/studio/view_components_2.gif)
-
 ### Filters:
 
-You can filter the commits that you want to display by the following fields:
-
-- **Branch:** The Git branch
-- **Tag:** The Git tag
-- **Author:** Author of the Git commit
-- **Metric:** Values of different metrics. For instance, you can display only
-  those experiments for which the value of `avg_prec` is greater than `0.9`.
-- **Metric delta:** Change in the value of the metric. For instance, you can use
-  this filter to only display those experiments for which the value of
-  `avg_prec` changed by more than `0.1` compared to the baseline experiment.
-- **Param:** Values of different parameters
-- **File size:** Size of the data, model and other files corresponding to your
-  experiments
-- **File changed:** Whether or not any given file changed in the experiment
+Click on the `Filters` button to specify which rows you want to show in the
+project table.
+
+![Project filters](https://static.iterative.ai/img/studio/project_filters.png)
+
+There are two types of filters:
+
+- **Quick filters** (highlighted in orange above): Use the quick filter buttons
+  to
+
+  - Show only DVC experiments
+  - Show only selected experiments
+  - Toggle hidden commits (include or exclude hidden commits in the project
+    table)
+
+- **Custom filters** (highlighted in purple above): Filter commits by one or
+  more of the following fields:
+
+  - Git related fields such as Git branch, commit message, tag and author
+  - Column values (values of metrics, hyperparameters, etc.) and their deltas
+
+  <admon type="info">
+
+  The `Branch`filter displays only the specified branch and its commits.
+
+  On the other hand, the `Commits on branch` filter displays all branches that
+  contain commits from the specified branch. For example, if a commit from the
+  `main` branch also appears in `feature-branch-1`, then filtering for
+  `Commits on branch = main` will include `main` as well `feature-branch-1` in
+  the project table.
+
+  </admon>
 
 ### Columns:
 
 Select the columns you want to display and hide the rest.
 ![Showing and hiding columns](https://static.iterative.ai/img/studio/show_hide_columns.gif)
 
-You can also click and drag the columns in the table to rearrange them.
-
 If your project is missing some required columns or includes columns that you do
 not want, refer to the following troubleshooting sections:
 
 - [Project does not contain the columns that I want](/doc/studio/troubleshooting#project-does-not-contain-the-columns-that-i-want)
 - [Project contains columns that I did not import](/doc/studio/troubleshooting#project-contains-columns-that-i-did-not-import)
 
+To reorder the columns, click and drag them in the table or from the Columns
+dropdown.
```

Comment:
```suggestion
dropdown:
```

Reply:
See [this comment](https://github.com/iterative/dvc.org/pull/4758#discussion_r1293408595).